### DISCUSSION TITLE
Why Quasar Vite CLI is targeting ES2019?

### DISCUSSION THREAD

**Discussion by Bullz3y3** [quasarframework/quasar#13737] [CLI - PWA mode]
Hi,
Can anyone explain why default target for Vite CLI is set to:

`browser: [ 'es2019', 'edge88', 'firefox78', 'chrome87', 'safari13.1' ]`

Is there a particular reason for selecting these? Furthermore, why there is no Opera, Android, and iOS in the selection?

**Comment by pdanpdan** [comment]
Opera is chrome, and android and iOS are not browsers.
And it needs to be a default - you can change it as you wish.
